### DISCUSSION TITLE
Configuring config.json for a Windows server for let's encrypt?

### DISCUSSION THREAD

**Discussion by boe1** [Ylianst/MeshCentral#5017] [Q&A] [answered]
Hello,

I tried to modify my config.json after watching the video on youtube.  I did something wrong and it would no longer start meshcentral.

If my URL for meshcentral for my company is support.mydomain.com  can someone post what the config file should be (assuming it was vanilla other than the let's encrypt part.

I belive this is the current one.
{
  "settings": {
    "cert": "support.mydomain.com",
    "_minify": true
  }
}

**Comment by boe1** [comment]
This is what I changed it to, loaded meschcentral and ran ran serverreset in the console but then meshcentral wouldn't load on my windows pc so I copied the old one back and rebooted.

{
  "settings": {
    "cert": "support.mydomain.com",
    "_minify": true,
    "port": 443,
    "redirectPort": 80
  },
  "letsencrypt": {
	"email":"me@mydomain.com",
	"names": support.mydomain.com",
	"production": false
  }
	 
}

**Comment by PathfinderNetworks** [comment, marked as answer]
You are missing a single comma after the LetsEncrypt code.
Should be like this:
{
"settings": {
"cert": "support.mydomain.com",
"_minify": true,
"port": 443,
"redirectPort": 80
},
"letsencrypt": {
"email":"[me@mydomain.com]",
"names": support.mydomain.com",
"production": false
},

}

**Reply by boe1** [reply]
Thanks!

**Reply by boe1** [reply]
Sorry - I should have posted the result - it had the same issue - would not load after the server reset.

**Comment by adimate71** [comment]
Hi,
Please let me know if it is possible to modify config.json on the Ubuntu server to use Cloudflare api? I would like to avoid opening port 80.
Thank you

**Comment by boe1** [comment]
Sorry - just figured it out - went to https://duckduckgo.com/ and put the script in!
